### PR TITLE
Edit getById function

### PR DESCRIPTION
**Pull request by ovekyc**
Edit get by id function 
Json which is sended by response now has imageUrls

Add fetchPrefix function

### CODE REVIEW

File: src/server/database.js [hunk 1]
```diff
@@ -20,6 +20,29 @@ export const clearDB = () => {
...
+  }).on('error', (err) => {
+    error = err;
+  }).on('close', () => {
+    if (error) {
+      return cb(error);
+    }
+    return cb(null, keys);
+  });
+};
+
```
Comment (hewonjeong) [inline src/server/database.js:45]: Good. It will be useful.

**Comment by festinalente91** [comment]
lgtm

**Comment by hewonjeong** [comment]
lgtm

**Comment by ovekyc** [comment]
LGTM

**Pull request merged**